core/src/main/res/values/strings.xml

```diff
@@ -472,6 +472,8 @@
     <string name="pref_gpodnet_notifications_sum">This setting does not apply to authentication errors.</string>
     <string name="pref_playback_speed_title">Playback Speeds</string>
     <string name="pref_playback_speed_sum">Customize the speeds available for variable speed audio playback</string>
+    <string name="pref_playback_time_respects_speed_title">Adjust media info to playback speed</string>
+    <string name="pref_playback_time_respects_speed_sum">Displayed position and duration are adapted to playback speed"</string>
```

Comment:
There is a `"` that accidentally got copied

Reply:
oups :-(